src/control/Control.Layers.js

```diff
@@ -417,6 +417,7 @@ export const Layers = Control.extend({
 	_expandIfNotCollapsed() {
 		if (this._map && !this.options.collapsed) {
 			this.expand();
+			document.querySelector('.leaflet-control-layers-list').setAttribute('aria-label', 'Layers');

```

Comment:
I feel like there must be a better way of doing this than querying the whole document. Is this element not stored somewhere as a field on this class already?

Reply:
That would be `this._section`:

https://github.com/Leaflet/Leaflet/blob/007025069d7afffc03c3ebd74dbf8bdc9ebe7cce/src/control/Control.Layers.js#L189

Reply:
Thank you @jonkoops and @Falke-Design! I added another commit to this so that it uses the `_section` class variable instead of querying the whole document. Also @Falke-Design I added another commit for your earlier comment on the case where it is expanded after being collapsed. Please let me know if this all looks okay, and if not, I'm happy to make more changes.